01week/rockPaperScissors.js

```diff
@@ -10,14 +10,47 @@ const rl = readline.createInterface({
 
 function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
+  //3. Compare User1 input to User2 input. (if else) (Combine hand1 and hand 2)
+  const combinedHands = hand1 + hand2;
+  //Create a switch statement that evaluates each case using the combined user inputs
+  switch (combinedHands) {
+    //If User1 input and User2 input are equal, it's a tie.
+    case 'rockrock':
+    case 'paperpaper':
+    case 'scissorsscissors':
+      console.log(`It's a tie.`);
```

Comment:
Nice use of the switch statement.  Be sure to return instead of writing to the console.